app/Console/Commands/GenerateEBConfig.php

```diff
@@ -74,6 +74,7 @@ public function getStubVariables(): array
     {
         return [
             'EB_ENV_NAME' => env('EB_ENV_NAME'),
+            'EB_ENV_DEV_NAME' => env('EB_ENV_DEV_NAME'),
```

Comment:
Seems to me we should be setting `EB_ENV_NAME` to a different value, not creating a new `DEV` version of the key -- and then having everything deploy based on that value.